### DISCUSSION TITLE
crash

### DISCUSSION THREAD

**Discussion by dakotx** [diesel-modding/PAYDAY-2-BeardLib#700] [General]
hello
everytime i download this beardlib
IT CRASHES ON BLACK CAT, BENEATH THE MOUNTAIN, THE WHITE HOUSE, BULUCS MANSION
BRO ARE YOU FREAKING LAZY?
Callstack:

         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     zip_error_init                                      
         payday2_win32_release  (???)     zip_error_init                                      
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
                      KERNEL32  (???)     BaseThreadInitThunk                                 
                         ntdll  (???)     RtlGetAppContainerNamedObjectPath                   
                         ntdll  (???)     RtlGetAppContainerNamedObjectPath                   

-------------------------------

Current thread: Main
Script stack:
        _serialize_to_script()  @mods/BeardLib/Classes/Managers/FileManager.lua:44  
_get_instance_continent_data()  core/lib/managers/coreworldinstancemanager.lua:242  
           prepare_unit_data()  core/lib/managers/coreworldinstancemanager.lua:250  
                    original()  core/lib/utils/dev/editor/coreworlddefinition.lua:303  
           _insert_instances()  @mods/base/req/core/Hooks.lua:264                   
            parse_continents()  core/lib/utils/dev/editor/coreworlddefinition.lua:286  
        WorldDefinition_init()  core/lib/utils/dev/editor/coreworlddefinition.lua:44  
        WorldDefinition_init()  @mods/BeardLib/Hooks/World/CoreWorldDefinition.lua:10  
                        init()  @mods/BeardLib/Hooks/World/CoreWorldDefinition.lua:10  
                        init()  lib/utils/dev/editor/worlddefinition.lua:6          
                         new()  core/lib/utils/coreclass.lua:35                     
                        init()  core/lib/utils/dev/editor/worldholder.lua:49        
                         new()  core/lib/utils/coreclass.lua:35                     
                   init_game()  lib/setups/gamesetup.lua:734                        
                                core/lib/setups/coresetup.lua:424                   

-------------------------------

System information:
	Application version : 1.143.230
	CPU : Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz (2 cores); SSE; SSE2; SSE3; SSSE3; SSE4.1; SSE4.2
	DirectX : 12.0 
	GPU : NVIDIA GeForce GTX 1050 Ti / nvldumd.dll[31.0.15.4584]
	Language : english
	Memory :     16339MB 120KB
	OS : 6.2.9200 () 0x300-0x1 (64 bits)
	Physics : threaded
	Renderer : DX9 threaded
	Sound : Realtek Semiconductor Corp. (HoparlÃ¶r (Realtek High Definition Audio))

**Comment by segabl** [comment]
Sounds like you have a broken mod_overrides mod

**Comment by dakotx** [comment]
How

21 Eki 2023 Cmt 21:11 tarihinde Sebastian Gabl (aka Hoppip) <
***@***.***> şunu yazdı:

**Comment by segabl** [comment]
remove all mod_overrides mods and see if you still crash
